internal/build/pom.xml

```diff
@@ -1424,23 +1430,25 @@
                     <name>jdbi.native</name>
                 </property>
             </activation>
+            <modules>
```

Comment:
this is now propagated to every pom inheriting from the build parent. This should not be here.